whylog/tests/tests_constraints/test_user_constraint.py

```diff
@@ -34,8 +34,8 @@ def test_verify_fail(self):
 
 class TestTimeConstraint(TestCase):
     def setUp(self):
-        self.min_delta = timedelta(seconds=1)
-        self.max_delta = timedelta(seconds=10)
+        self.min_delta = 1
```

Comment:
We want that yaml files still be readable and editable by human. So I can't serialize timedelta object. We should create constraint's classes, which make this conversion once in constructor. Now constraints in rule is a list of dictionaries. For no block maybe we should leave it in this form.